### DISCUSSION TITLE
Hooks to run on new certificate

### DISCUSSION THREAD

**Discussion by rithvikvibhu** [smallstep/certificates#760] [Q&A] [answered]
I want to run a script / do some work for every issued certificate, with access to the certificate and email (to generate and send TLSA records, but this is out of scope).

Looked through the docs and GitHub issues and didn't find anything.

**Comment by tashian** [comment, marked as answer]
We don't have a specific feature in the CA for this.
You could have something follow the CA logs and use that to build a hook.
When a certificate is issued, the entire base64 encoded cert contents are logged.

**Reply by rithvikvibhu** [reply]
That's too bad, step-ca is perfect otherwise. I'll look into options (logs or alternatives). Thanks for the reply!
